### ISSUE TITLE
Custom generation settings says size in chunks when the actual behavior is size in blocks

### ISSUE THREAD

**Issue by SuperiorDonut** [quiqueck/BetterEnd#80]
### What happened?

![BEG](https://user-images.githubusercontent.com/114554928/192689447-faf89644-e4d6-4439-9177-1c7366c8ac96.png)
![Result](https://user-images.githubusercontent.com/114554928/192689446-338e6706-e2e8-4e5d-ba70-3e1b4cb462a4.png)

When in the "BetterX world generation settings" screen, above the biome size sliders it says "average biome size (in chunks)." The actual result is biomes being generated with the sliders representing biome size in blocks (verified by mod author on their discord); proof of the result can been seen in the attached images.

### BetterEnd

2.1.0

### BCLib

2.1.0

### Fabric API

0.61.0

### Fabric Loader

0.14.9

### Minecraft

1.19

### Other Mods

```shell
Biome Makeover (by Lemonszz)
Entity Texture Features - [Fabric & Forge] (by traben_0)
DeepslateCutting (by NoComment1105)
Cloth Config API (Fabric/Forge) (by shedaniel)
DragonLoot (by pois1xlive)
Charmonium (by svenhjol)
Bountiful (Fabric) (by Ejektaflex)
Transparent (by Trikzon)
AdventureZ (by Globox_Z)
Creatures From The Snow! [Fabric] (by YoWabbit)
DimensionalDoors (by DimensionalDevelopment)
Terrestria (by coderbot)
Eroding Stone Entities (Fabric) (by Serilum)
Colormatic (by kwertiTheCats)
OptiGUI (by opekope2)
Patchouli (Fabric/Quilt) (by williewillus)
Audaki Cart Engine: Faster Minecarts / Balanced for Survival / Vanilla Carts Improved (by Audaki_kira)
Nears [Fabric] (by DigitalPear_11)
MoreMcmeta (Fabric) (by soir20)
Diagonal Fences (by Fuzs_)
xali's Enchanted Books (by xalixilax)
Bartering Station (by Fuzs_)
Better Tridents (by Fuzs_)
Ellemes' Container Library (by 3llemes)
Better Compatibility Checker (by Gaz_)
Inventory HUD+ (by dmitrylovin)
Farmer's Knives [Fabric] (by ianm1647)
Forge Config API Port (by Fuzs_)
Cardinal Components (by UpcraftLP)
Balm (Fabric Edition) (by BlayTheNinth)
Effective 💦 (by doctor4t)
Custom Entity Models (CEM) (by dorianpb)
LazyDFU (by tuxed)
Charm of Undying (Fabric) (by TheIllusiveC4)
Stoneholm, Underground Villages (Fabric) (by TheGrimsey)
AttributeFix (by DarkhaxDev)
SlimyFloor (by Pouille)
Comforts (Fabric) (by TheIllusiveC4)
[Fabric] Extended Armor Bars Forked (by Slugstur)
Starlight (Fabric) (by Spottedstar)
FancyMenu [Fabric] (by Keksuccino)
Konkrete [Fabric] (by Keksuccino)
Sodium (by jellysquid3_)
Quartz Elevator (by atonkish)
MVS - Moog's Voyager Structures (by finn3253)
More Villagers [Fabric] (by SameDifferent)
Controlling (by Jaredlll08)
Bow Infinity Fix (Forge/Fabric) (by Parker8283)
Animatica (by FoundationGames)
XP Storage (by sfinc)
Enchantment Descriptions (by DarkhaxDev)
PolyLib (by Official_CreeperHost)
When Dungeons Arise - Fabric! (by Aureljz)
Naturalist (by Starfish_Studios)
Fabric Seasons (by D4rkness_King)
Smooth Boot (Fabric) (by UltimateBoomer)
Drippy Loading Screen [Fabric] (by Keksuccino)
BisectHosting Server Integration Menu [FABRIC] (by BisectHosting)
Open Loader (by DarkhaxDev)
Elytra Slot (Fabric) (by TheIllusiveC4)
Illuminations 🔥 (by doctor4t)
LambDynamicLights (by LambdAurora)
Pling (by haykam)
Item Filters (by LatvianModder)
DashLoader (by notequalalpha)
Farmer's Delight [Fabric] (by NewHoryzon)
Tom's Simple Storage Mod (Fabric) (by tom54541)
Craft the Totem of Undying [FORGE & FABRIC] (by 00theblackwolf00)
BetterEnd (by Quiqueck)
Entity Culling Fabric/Forge (by tr9zw)
Smelting sand in blast furnace (by 1_block_microwave)
Beacon Overhaul (by HeckinChloe)
Xaero's Minimap - Modded Support (by babybluetit)
Architect's Palette [FABRIC] (by slomax907)
BCLib (by Quiqueck)
Carrier (by GabrielHOlv)
Wool Tweaks (Fabric) (by Serilum)
Not Enough Animations (by tr9zw)
KleeSlabs (Fabric Edition) (by BlayTheNinth)
On Soul Fire (by MoriyaShiine)
Server Performance - Smooth Chunk Save[Forge/Fabric] (by someaddon)
End Goblin Traders (Fabric) (by jab125twitchy)
Eden Ring (by paulevs)
Blur (Fabric) (by Motschen)
Redstone Pen (by wilechaote)
Roughly Enough Professions (REP) (by Mrbysco)
SkyLib (by SkylorBeck)
Void Totem (Fabric) (by Affehund)
Nature's Compass (by Chaosyr)
Indium (by comp500)
Mod Menu (by ProspectorDev)
🔨 Frame API (by andantet)
Repurposed Structures (Quilt/Fabric) (by telepathicgrunt)
Villager Death Messages (Fabric) (by Serilum)
Oxidized (by safrodev)
Oh The Biomes You'll Go Refabricated (by AOCAWOL)
Redstone Bits (by Shnupbups)
LambdaBetterGrass (by LambdAurora)
Goblin Traders (Fabric) (by jab125twitchy)
RightClickHarvest (by jamalam360)
YetAnotherConfigLib (by XanderIsDev)
Village Bell Recipe (Fabric) (by Serilum)
XP Storage - Trinkets (by sfinc)
Luggage (by GizmoTheMoonPig)
Additional Structures (Fabric) (by XxRexRaptorxX)
Fabric Language Kotlin (by modmuss50)
JamLib (by jamalam360)
ThonkUtil (by jab125twitchy)
Tips (by DarkhaxDev)
Incendium (by Starmute)
Inventory Totem (Fabric) (by Serilum)
Chalk (Fabric) (by DaFuqsy)
Collective (Fabric) (by Serilum)
Extra Alchemy (by zabi94)
Bosses of Mass Destruction (by Barribob)
Easy Magic (by Fuzs_)
Pyrotastic 🎆 (by creativityshark)
Charm (by svenhjol)
Dawn API (by Hugman_76)
Traveler's Backpack [Fabric] (by Tiviacz1337)
Xaero's World Map (by xaero96)
Fabric API (by modmuss50)
FTB Backups 2 (by FTB)
GeckoLib (by ThanosGecko)
mine11lib (by itsmineblock11)
Roughly Enough Resources (by theorbagain)
Stay True (by haimcyfly)
Just an End Anchor (by KxmischesDomi)
Icy Incitement (by AmyMialee)
Fixed Anvil Repair Cost (Fabric) (by Serilum)
Starter Kit (Fabric) (by Serilum)
Trinkets (Fabric) (by EmilyPloszaj)
Zoomify (by XanderIsDev)
Visuality (by PinkGoosik)
The Below Bedrock Caves (by MrCrazyMan109)
Another Furniture (by Starfish_Studios)
Default Dark Mode (by nebuIr)
More Axolotl Variants Mod (by Akashii_Kun_)
Better Statistics Screen (Fabric) (by TheCSDev)
Chat Heads (by dzwdz)
Decorative Blocks (by stohun)
Axolotl Bucket Fix (by colderlavalamp)
Expanded Storage (by 3llemes)
Conduits Prevent Drowned (Fabric) (by Serilum)
End Portal Recipe (Fabric) (by Serilum)
FTB Library (Fabric) (by FTB)
FTB Quests (Fabric) (by FTB)
FTB Teams (Fabric) (by FTB)
Chubby Stuff! [Fabric] (by YoWabbit)
Universal Ores (Fabric) (by Hugman_76)
Inmis (by Draylar1)
Puzzles Lib (by Fuzs_)
Blockus (by Brandcraft_)
Wolves With Armor (by Draylar1)
Bookshelf (by DarkhaxDev)
Trading Post (by Fuzs_)
JumpOverFences (by Martmists)
Spirit (by CodexAdrian)
Tool Stats (by DarkhaxDev)
Moonlight Lib (by MehVahdJukaar)
Friends&Foes (Fabric) (by faboslav)
Nullscape (by Starmute)
Crying Ghasts (Fabric) (by Serilum)
Equipment Compare [Fabric] (by Grend_G)
Crawl (Fabric) (by fewizz_)
Roughly Enough Items Fabric/Forge (REI) (by shedaniel)
Pick Up Notifier (by Fuzs_)
Cloth Config API (UNUSED) (by shedaniel)
Paintings ++ (by AbsolemJackdaw)
No Chat Reports (by Aizistral)
Better Conduit Placement (Fabric) (by Serilum)
Straw Golem Rebaled (by CommodoreThrawn)
Better Bad Omen (by PrussianBIue)
CleanCut (by Rongmario)
More Babies (Forge & Fabric) (by mammut53m)
Adorn (by Juicebus)
TerraBlender (Fabric) (by TheAdubbz)
Lithium (Fabric) (by jellysquid3_)
FerriteCore (Fabric) (by malte0811)
Duckling (by Frogipher)
Piercing Paxels (by AmyMialee)
Continuity (by Pepper_Bell)
bad packets (by badasintended)
Every Compat (by MehVahdJukaar)
spark (by Iucko)
Ecologics (by SameDifferent)
AppleSkin (by squeek502)
Architectury API (Fabric/Forge) (by shedaniel)
Expanded Delight [Fabric] (by ianm1647)
Reese's Sodium Options (by FlashyReese)
Cherished Worlds (Fabric) (by TheIllusiveC4)
BetterNether (by Quiqueck)
Dark Paintings (by DarkhaxDev)
Magnum Torch (by Fuzs_)
Villager Names (Fabric) (by YumatanGames)
Minecraft Transit Railway (by jonafanho)
Debugify (by XanderIsDev)
Falling Leaves (Fabric) (by RandomMcSomethin)
Simple Discord Rich Presence (Forge / Fabric) (by Sunekaer)
Sodium Extra (by FlashyReese)
WTHIT (by badasintended)
Dwarf Coal (Fabric) (by traverse_joe)
Universal Bone Meal (by Fuzs_)
Mouse Tweaks (by YaLTeR)
Wolves Of Other Furs (W.O.O.F) (by itsmineblock11)
Iris Shaders (by coderbot)
Creeper Overhaul (by joosh_7889)
Polymorph (Fabric) (by TheIllusiveC4)
Cull Less Leaves (by XanderIsDev)
CIT Resewn (by SHsuperCM)
The Bumblezone (Quilt/Fabric) (by telepathicgrunt)
Clumps (by Jaredlll08)
Xaero's Minimap (by xaero96)
MemoryLeakFix (by FX_PR0CESS)
You're in Grave Danger (by b1n_ry)
Fish On The Line (Fabric) (by Serilum)
Windchimes (by hibiscvs)
InmisAddon (by Globox_Z)
Item Highlighter [Fabric] (by Grend_G)
Auudio [Fabric] (by Keksuccino)
Skin Layers 3D (Fabric/Forge) (by tr9zw)
Fabric Waystones (by LordDeatHunter)
🪵 Twigs (by ninnih_)
More Geodes (by TheDeathlyCow)
Kambrik (by Ejektaflex)
Structory (by botanydev)
Visual Workbench (by Fuzs_)
Iceberg [Fabric] (by Grend_G)
Puzzle (by Motschen)
Grim's Transportables (by TheGrimsey)
Supplementaries (by MehVahdJukaar)
Tokenable Furnaces and Storage (by SkylorBeck)
Item Model Fix (Fabric) (by Pepper_Bell)
Open Parties and Claims (by xaero96)
FallingTree (Forge&Fabric) (by RakSrinaNa)
Chalk (Fabric): Colorful Addon (by DaFuqsy)
VoidZ (by Globox_Z)
MidnightLib (by TeamMidnightDust)
Chisel Refabricated (by Slaincow)
Mo' Colors (by GabrielHOlv)
Better Advancements (by way2muchnoise)
Vanilla Excavators (by Draylar1)
Go Fish (by Draylar1)
Deeper And Darker (by KyaniteMods)
Goodall (by Frogipher)
Krypton (by tuxed)
Galosphere (by orcinus73)
```

**Comment by MCCrafter247** [comment]
I can confirm this happens even when Better End 2.1.0, fabric api 0.62.0, and bclib 2.1.0 are the only mods present. I played it on Minecraft 1.19.2 and fabric loader 0.14.9. It must just be mistake somewhere.
![unknown](https://user-images.githubusercontent.com/108512960/193627770-c96c907f-28c7-4e73-a19d-5011987b0428.png)

**Comment by quiqueck** [comment]
Thanks for reporting. Will be fixed in the next BCLib Release.

**Comment by quiqueck** [comment]
Fixed in BCLib 2.1.1
